inc/about-us/section-5.inc

```diff
@@ -0,0 +1,17 @@
+<div class="container-fluid">
+  <div class="row">
+
+    <div class="col">
+    
+    </div>
+
+    <div class="col">
+    
+    </div>
+
+    <div class="col">
+    
+    </div>
+
+  </div>
+</div>
```

Comment:
Please add newline at the end of the file.

Reply:
ready